### PR TITLE
Add OCM 3 tournament

### PR DESCRIPTION
**Pull request by TPGPL**
tracks #2933 with significant changes

### CODE REVIEW

File: wiki/Tournaments/OCM/3/en.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1 month of osu!supporter tag |
+| ![MVP icon](img/mvp.png "MVP") | 1 month of osu!supporter tag |
+
+*Note: The MVP is chosen by the tournament staff after the tournament concludes.*
+
+## Organisation
+
+The osu! Cup Master 3: osu!catch edition tournament is run by various osu! community members.
+
+| Position | Members |
```
Comment (fajar13k) [inline wiki/Tournaments/OCM/3/en.md:39]:
```suggestion
| Position | Member(s) |
```
One position only taken by 1 person, maybe necessary

File: wiki/Tournaments/OCM/3/en.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+| Streamer | ![][flag_MX] [daxeroz](https://osu.ppy.sh/users/1170156), ![][flag_LU] [Gae](https://osu.ppy.sh/users/7787785) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/997764)
+- [Livestream](https://www.twitch.tv/osucupmaster)
+- [Discord server](https://discord.gg/Y9MKD8m)
+- [Twitter](https://twitter.com/CupOsu)
+
+---
```
Comment (fajar13k) [inline wiki/Tournaments/OCM/3/en.md:56]:
```suggestion

```
I think it's better to delete these line breaks. (Apply for all LBs)

File: wiki/Tournaments/OCM/3/en.md [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+
+## Ruleset
+
+### Tournament rules
+
+1. This version of the OCM is a team tournament, played on the osu!catch game mode.
+2. This competition is a 2 versus 2 setup. The team size can be up to 4 members, with a minimum of 3 members per team.
+3. Score V2 will be used.
+4. The tournament is open to any player with a rank of at least #4000. There's no restriction of country or region.
```
Comment (fajar13k) [inline wiki/Tournaments/OCM/3/en.md:246]:
```suggestion
4. The tournament is open to any player with a rank of at least #4,000. There's no restriction of country or region.
```

File: wiki/Tournaments/OCM/3/en.md [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+4. Following stages are Double Elimination Stages. This means that the winner moves to the next stage and the losing team gets moved to the Loser bracket.
+5. If a team loses a match on the loser bracket, they are eliminated from the tourney.
+
+**Winning conditions:**
+
+- In Group Stage, you need to win 5 maps to win a match. (Best-of-9)
+- In the Quarterfinals and the Semifinals, you need to win 6 maps to win a match. (Best-of-11)
+- In the Finals and Grand Finals, you need to win 7 maps to win the match. (Best-of-13)
+
+[flag_MX]: /wiki/shared/flag/MX.gif
```
Comment (fajar13k) [inline wiki/Tournaments/OCM/3/en.md:313]:
```suggestion
[flag_MX]: /wiki/shared/flag/MX.gif
```
No flag descs (?)

File: wiki/Tournaments/OCM/3/en.md [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+tags:
+- OCM 3
+- OCM3
+---
+
+# osu! Cup Master 3: osu!catch edition
+
+![OCM3 logo](img/logo.png)
+
+The **osu! Cup Master 3: osu!catch edition** (***OCM3***) is a a two-on-two team based osu!catch tournament hosted by ![][flag_PA] [Jafett](https://osu.ppy.sh/users/2317102) and ![][flag_MX] [daxeroz](https://osu.ppy.sh/users/1170156). It is the 3rd installment of the osu! Cup Master.
```
[the remark this answers is not in the document]
Comment (TPGPL) [reply wiki/Tournaments/OCM/3/en.md:11]: not gonna conclude the tournament as it wasnt part of the original PR, + i have no data on podium, mappools etc. ¯\_(ツ)_/¯

**Pull request merged**